build-scripts/for-linux/prepare-debian-for-tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Install requirements for tests infrastructure
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y git gcovr
+
+# Install cpptrace from sources
+mkdir cpptrace
+cd cpptrace
+git clone https://github.com/jeremy-rifkin/cpptrace.git .
```

Comment:
May be unpack from release https://github.com/jeremy-rifkin/cpptrace/archive/refs/tags/v0.4.1.tar.gz instead of cloning from git?

Reply:
Yes, maybe to avoid installing git

Reply:
Done